From b6f5471acefd673e5146f239e624009482e665de Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Fri, 12 Jan 2024 14:36:39 +0100
Subject: [PATCH 0877/1141] arm64: dts: st: update hdmi node on stm32mp257f-ev1
 board

Rework power supplies of hdmi bridge to insert regulator V1V8.

Change-Id: Ie20e0b0339c52a51ae2c0e2c555e338e7be19606
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/350986
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index 7ba627b7d832..ff8c90d6301c 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -332,12 +332,12 @@ adv7535: hdmi@3d {
 		status = "disabled";
 		adi,dsi-lanes = <4>;
 		reset-gpios = <&gpiog 14 GPIO_ACTIVE_LOW>;
-		avdd-supply = <&scmi_v3v3>;
-		dvdd-supply = <&scmi_v3v3>;
-		pvdd-supply = <&scmi_v3v3>;
-		a2vdd-supply = <&scmi_v3v3>;
+		avdd-supply = <&scmi_v1v8>;
+		dvdd-supply = <&scmi_v1v8>;
+		pvdd-supply = <&scmi_v1v8>;
+		a2vdd-supply = <&scmi_v1v8>;
 		v3p3-supply = <&scmi_v3v3>;
-		v1p2-supply = <&scmi_v3v3>;
+		v1p2-supply = <&scmi_v1v8>;
 
 		ports {
 			#address-cells = <1>;
-- 
2.39.2

